src/components/cards/BiobankReportCard.vue

```diff
@@ -1,210 +1,119 @@
 <template>
-  <div v-if="biobank.data.collections">
-    <router-link :to="{path: '/biobankexplorer', query: query}">Back to search results</router-link>
-
-    <div class="card biobank-card">
-      <div class="card-header">
-        <div class="row">
-          <div class="col-md-10">
-            <h3>{{biobank.data.name}}</h3>
-
-            <small>
-              <dl class="row">
-                <dt class="col-sm-3"><b>Collection types:</b></dt>
-                <dd class="col-sm-9"><i>{{ collectionTypes }}</i></dd>
-                <dt class="col-sm-3"><b>Number of collections:</b></dt>
-                <dd class="col-sm-9"><i>{{ biobank.data.collections.length }}</i></dd>
-                <dt class="col-sm-3"><b>Number of samples:</b></dt>
-                <dd class="col-sm-9"><i>{{ numberOfSamples }}</i></dd>
-              </dl>
-            </small>
-
-
-            <p>{{biobank.data.description}}</p>
+  <div class="mg-biobank-card container">
+    <loading
```

Comment:
Have you considered putting this in App.vue?
Less duplication and I'd think it is probably useful everywhere?
Or did it look bad in the explorer?